scripts/mongo/startDiscoverWorkers.sh

```diff
@@ -1,3 +1,14 @@
+#!/usr/bin/env bash
 # development only.
```

Comment:
what would happen if someone tried to run these scripts on production?
is there anything we could use in the environment to make these script exit if they are run on anything other than development?